frontend/src/components/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -122,18 +124,37 @@ const PlayersList = (props: PlayersListProps) => {
   );
 };
 
-const PlayerNotFound = () => (
-  <Centered>
-    <Alert severity="info">
-      <AlertTitle>Nie znaleziono zawodnika?</AlertTitle>
-      Zapisz nowego poniżej.
-    </Alert>
-    <VerticalSpace height="1rem" />
-    <Button variant="contained" color="primary">
-      Dodaj i zapisz
-    </Button>
-  </Centered>
-);
+const PlayerNotFound = () => {
+  const [drawerState, setDrawerState] = useState<boolean>(false);
+
+  const toggleDrawer = (open: boolean) => () => {
+    setDrawerState(open);
+  };
+
+  return (
+    <Centered>
```

Comment:
Boje się trochę tego Centered / VerticalSpace. Ja to zrobiłem tak "na szybko", a teraz wszędzie tego używacie. Żeby potem nie było, że nie mówiłem - w przypadku jakiś problemów z responsywnością :)

Reply:
no fakt, tutaj wtedy może być problem :-| - pytanie w takim razie jak to zamienić tak, żeby było dobrze na gorsze czasy?
@PiotrWR